site/content/en/reference/live/_index.md

```diff
@@ -20,4 +20,129 @@ description: >
 
 Live contains the next-generation versions of apply related commands for
 deploying local configuration packages to a cluster.
+
+### How to Upgrade to the Next Generation Inventory Object
+
+#### What is an Inventory Object
+
+An inventory object is the automatically generated object which keeps track
+of the set of objects applied together. The current inventory object type
+is a ConfigMap, and it is usually defined in a package file called
+**inventory-template.yaml**. This file is created from an invocation of
+`kpt live init`. A typical use of the inventory object is to prune (delete)
+objects omitted locally.
+
+#### What is a Next Generation Inventory Object
+
+The next generation inventory object is a **ResourceGroup** custom resource
+replacing the current ConfigMap. Because the new inventory object is a
+custom resource, you must have permissions to add a custom resource
+definition (CRD) to the cluster. The individual command to add the
+**ResourceGroup** CRD is:
+
+```
+export RESOURCE_GROUP_INVENTORY=1
```

Comment:
Given we forked a `next` branch, do we still want to support both mechanisms in the same version of kpt CLI? Instead we can say that version N-1 uses ConfigMap and version N uses ResourceGroup.

Reply:
Until we force customers off the legacy ConfigMap inventory object by automatically migrating them, we must allow them to run `kpt live` on these packages which the legacy mechanism. I do not believe we will be at that point (Phase Four) for at least six months, and probably longer.